sys/align.go

```diff
@@ -31,6 +36,88 @@ func initAlign() {
 	}
 }
 
+func markBitfieldLast(t *Type) {
+	switch t1 := (*t).(type) {
+	case *IntType:
+		t1.BitfieldLst = true
+	case *ConstType:
+		t1.BitfieldLst = true
+	case *LenType:
+		t1.BitfieldLst = true
+	case *FlagsType:
+		t1.BitfieldLst = true
+	case *ProcType:
+		t1.BitfieldLst = true
+	default:
+		panic(fmt.Sprintf("type %+v can't be a bitfield", t1))
+	}
+}
+
+func setBitfieldOffset(t *Type, offset uintptr) {
```

Comment:
No need to pass Type by pointer. It already contains a pointer to concrete type, so *Type is a pointer to pointer. Just pass Type.
Same for markBitfieldLast.